beta-src/src/webDiplomacyTheme.ts

```diff
@@ -13,6 +13,21 @@ declare module "@mui/material/styles" {
     desktop: true;
   }
 
+  interface CountryColor {
+    main: string;
+    light: string;
+  }
+
+  interface PaletteOptions {
+    France: CountryColor;
+    Austria: CountryColor;
+    England: CountryColor;
+    Germany: CountryColor;
+    Russia: CountryColor;
+    Italy: CountryColor;
+    Turkey: CountryColor;
+  }
```

Comment:
create an Enum for the countries and put them in src/enums, then use the enum here. Give PaletteOptions a type forcing it use the enum as a keys. Types should be added to src/types and then imported.